### DISCUSSION TITLE
Command and Option in help text do not align when name is long

### DISCUSSION THREAD

**Discussion by wayneyaoo** [Tyrrrz/CliFx#113] [Q&A] [answered]
Hi,

First thanks for making such great project! I had a question when i was using this. When the parameter name or option name or the command name was long, its description doesn't along with other descriptions. Is there anyway I can fix this? 

 e.g.
![image](https://user-images.githubusercontent.com/20643897/144707747-e2fd569b-7c0a-4fe2-98f6-4cc9c06e1af3.png)

I though all option descriptions would be aligned automatically. Is it possible?

**Comment by Tyrrrz** [comment, marked as answer]
Yeah, they have a minimum padding and then if the name is too long, it gets padded further away. I found that if everything is aligned the same, then the shorter options look awkward with too much empty space.

**Reply by wayneyaoo** [reply]
Yeah that's my guess too. but generally people wouldn't make the long option too long to "awkward"  those short ones right lol. So from probability point of view, can we e.g., at least increase the minimal padding or make the padding be within a range so that unless it's long enough to make short ones feel weird, it wouldn't break the text.

**Reply by Tyrrrz** [reply]
I guess we can introduce some kind of help configuration

**Reply by fossbrandon** [reply]
I looked into how other CLI tools handle this and all the ones I looked at appear to pad the descriptions evenly past the longest name. To me, it would make sense to default to consistent padding even if it does push out descriptions longer than ideal to follow what seems like the CLI norm. It wouldn't hurt to make it configurable in case people want to change whatever the default behavior is. If this gets converted to an issue and specced out, I could volunteer to try and implement the change.

Examples:

Docker CLI
![image](https://user-images.githubusercontent.com/46940941/162682186-c7f15dca-bf6f-4741-b0af-6b9caa68d7ce.png)

Winget CLI
![image](https://user-images.githubusercontent.com/46940941/162682301-f18e07f1-e450-4db2-8e9a-7f695f3bb53e.png)
